Extension/package.json

```diff
@@ -1032,6 +1032,7 @@
     "@types/mocha": "^2.2.43"
   },
   "dependencies": {
+    "jsonc-parser": "^1.0.0",
```

Comment:
Wow -- you found a jsonc parser. We actually want to use that for our parsing of c_cpp_properties.json, because currently comments are not allowed in that file. You need to get this added to our ThirdPartyNotices.txt.